src/objects/core/models.py

```diff
@@ -122,7 +122,9 @@ class ObjectRecord(models.Model):
         blank=True,
         null=True,
         help_text=_(
-            "Point, linestring or polygon object which represents the coordinates of the object"
+            "Point, linestring or polygon object which represents the coordinates of the "
+            "object. Geometry can be added only if the related OBJECTTYPE allows this "
+            "(`OBJECTTYPE.hasGeometry = true`)"
```

Comment:
Or doesn't exist...

Reply:
Do you want me to change back the description?

Reply:
the description is updated